src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
 
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number
 }
 
-export class Circle {
+const isValid = (...items: number[]): void => {
+  if (items.some((num) => num <= 0)) {
+    throw new Error('Params should be more than 0');
+  }
+};
+
+const rounder = (num: number): number => {
+  return Math.floor(num * 100) / 100;
+};
+
+export class Triangle implements Figure {
+  shape = Shape.Triangle;
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
It is bad practice to make fields public when unnecessary. This is a part of the encapsulation OOP principle

Reply:
fixed